uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -352,6 +364,9 @@ public void messageArrived(String topic, MqttMessage mqttMessage) {
                 userProps.forEach(p -> logger.atInfo()
                         .log("Received MQTT userProperties: {}, {}", p.getKey(), p.getValue()));
             }
+            if (contentType != null) {
+                logger.atInfo().log("Received MQTT message has 'content type' {}", contentType);
```

Comment:
We need take on quote not fixed test but value of content type which can contains spaces.

Like:
```
logger.atInfo().log("Received MQTT message has content type '{}'", contentType);
```

Without that result can be 
```
Received MQTT message has content type text/plan; Content-Encoding: utf-8
```
and its hard to identify what is a value.
Compare to 
```
Received MQTT message has content type 'text/plan; Content-Encoding: utf-8'
```